tensorflow_serving/model_servers/server.cc

```diff
@@ -182,8 +182,8 @@ void Server::PollFilesystemAndReloadConfig(const string& config_file_path) {
 }
 
 Status Server::BuildAndStart(const Options& server_options) {
-  if (server_options.grpc_port == 0) {
-    return errors::InvalidArgument("server_options.grpc_port is not set.");
+  if (server_options.grpc_port <= 0 && server_options.grpc_socket_path.empty()) {
+    return errors::InvalidArgument("one of server_options.grpc_port/server_options.grpc_socket_path should be set.");
```

Comment:
change message to:

"At least one of server_options.grpc_port or server_options.grpc_socket_path must be set."